src/Mutator/Boolean/LogicalOr.php

```diff
@@ -51,6 +55,7 @@
 final class LogicalOr implements Mutator
 {
     use GetMutatorName;
+    private ?string $seenVariabeName = null;
```

Comment:
```suggestion

    private ?string $seenVariabeName = null;
```

I wish PHP-CS-Fixer would pick that up but I can't see the rule that does that

Reply:
`class_attributes_separation`